CHANGELOG.md

```diff
@@ -3,6 +3,93 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 7.12.0
+
+_Aug 1, 2024_
+
+We'd like to offer a big thanks to the 12 contributors who made this release possible. Here are some highlights ✨:
+
+<!-- Blogpost for price changes, use it on tree view pro announcement -->
+
+[Upcoming changes to MUI X pricing in 2024](https://mui.com/blog/mui-x-sep-2024-price-update/)
+
+- 📦 [TreeView] Support item reordering using drag and drop (#12213)
```

Comment:
```suggestion
- 📦 [TreeView] Support item reordering using drag and drop (#12213)

  https://github.com/user-attachments/assets/64d926e2-d1b9-400e-a0b2-fde5ae3fac3f

```

Reply:
For me in the highlight we don't link the name of the PR and we don't put the prefix